src/synthetic.js

```diff
@@ -805,10 +807,24 @@ extend(syn.init.prototype, {
 	 */
 	"_click": function (element, options, callback, force) {
 		syn.helpers.addOffset(options, element);
+		if(syn.support.pointerEvents){
+			syn.trigger(element, 'pointerdown', options);
+		}
+		if(syn.support.touchEvents){
+			syn.trigger(element, 'touchstart', options);				
+		}
+		
 		syn.trigger(element, "mousedown", options);
 
 		//timeout is b/c IE is stupid and won't call focus handlers
 		schedule(function () {
+			if(syn.support.pointerEvents){
```

Comment:
change looks good to me, having the tests demostrate the cases of pointer, touch supported and not supported will be nice